### DISCUSSION TITLE
No VCalendar in v3?

### DISCUSSION THREAD

**Discussion by lookto** [vuetifyjs/vuetify#16030] [General] [answered]
As far as I can see in the new docs the calendar component is not supported in version 3.

Are there any information about if its still WIP or if it's deprecated and removed?

**Comment by KaelWD** [comment, marked as answer]
#13469, #15038

**Reply by lookto** [reply]
Thanks a a lot. I was not aware of the issues used for tracking tasks and only searched the discussions and docs.

Would it be a good idea to add a note to the v3 docs for all components that are in v2 and will be added in v3.x.x? So people looking into it coming from v2 don't think the components are deprecated in v3?
